Providers/Profile.swift

```diff
@@ -233,6 +233,7 @@ public class BrowserProfile: Profile {
 
         if isChinaEdition {
             prefs.setString(PrefsDefaults.ChineseHomePageURL, forKey: PrefsKeys.KeyDefaultHomePageURL)
+            prefs.setBool(false, forKey: PrefsKeys.KeyHomePageButtonIsInMenu)
```

Comment:
I see that this is set on each profile init if `isChinaEdition` is `true`. Doesn't that mean that if a Chinese user tries to change their share menu setting, their choice will just be clobbered the next time the app starts?